packages/modelviewer.dev/examples/tester.html

```diff
@@ -0,0 +1,227 @@
+    }
+
+    /* Nokia 8810 */
+
+    @media screen and (max-width: 320px) {
+      #instructions {
+        display: none;
+      }
+    }
+  </style>
+</head>
+<body>
+
+  <div id="container">
+    <div id="instructions">
+    <h1>
+      <a href="https://github.com/GoogleWebComponents/model-viewer" target="_blank">&lt;model-viewer&gt;</a>
+    </h1>
+    Drop a <a href="https://github.com/KhronosGroup/glTF" target="_blank">GLB or zipped GLTF</a> for the model =><br/>
+    Drop an <a href="https://hdrihaven.com/" target="_blank">HDR or JPG</a> for the skybox =><br/>
+    <input type="file" id="input" multiple><br/><br/>
+
+    <label for="src">src:</label>
+    <select id="src">
+      <option value="../shared-assets/models/Astronaut.glb">Astronaut</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/BoomBox/glTF-Binary/BoomBox.glb">Boom Box</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/BrainStem/glTF-Binary/BrainStem.glb">Brain Stem</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/Corset/glTF-Binary/Corset.glb">Corset</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/DamagedHelmet/glTF-Binary/DamagedHelmet.glb">Damaged Helmet</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/FlightHelmet/glTF/FlightHelmet.gltf" selected>Flight Helmet</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/Lantern/glTF-Binary/Lantern.glb">Lantern</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/Suzanne/glTF/Suzanne.gltf">Monkey</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/SpecGlossVsMetalRough/glTF-Binary/SpecGlossVsMetalRough.glb">Water Bottles</option>
+      <option value="../shared-assets/models/RobotExpressive.glb">Robot Expressive</option>
+      <option value="../shared-assets/models/alpha-blend-litmus.glb">Transparency Test</option>
+      <option value="../shared-assets/models/glTF-Sample-Models/2.0/MetalRoughSpheres/glTF/MetalRoughSpheres.gltf">Metal Rough Spheres</option>
+    </select><br/>
+    
+    <label for="environmentImage">environment-image:</label>
+    <select id="environmentImage">
+      <option value="" selected>default</option>
+      <option value="../shared-assets/environments/spruit_sunrise_1k_HDR.hdr">spruit sunrise HDR</option>
+      <option value="../shared-assets/environments/spruit_sunrise_1k_LDR.jpg">spruit sunrise JPG</option>
+      <option value="../assets/music_hall_01_1k.hdr">music hall</option>
+      <option value="../assets/whipple_creek_regional_park_04_1k.hdr">whipple creek</option>
+      <option value="../shared-assets/environments/aircraft_workshop_01_1k.hdr">aircraft workshop</option>
+      <option value="../assets/pillars_1k.hdr">pillars</option>
+    </select><br/>
+
+    <label for="useSkybox">use as skybox:</label>
+    <input id="useSkybox" type="checkbox"><br/>
```

Comment:
Checking this when `environmentImage` is "default" doesn't do anything. I would suggest either disabling the checkbox in that case (add the `disabled` attribute) or backfilling with the actual IBL we have.